package.json

```diff
@@ -25,8 +25,9 @@
     "phoenix:ci": "npm run phoenix -- --no-optional --no-fund --no-audit",
     "release": "sui-mono release",
     "start": "sui-studio start",
-    "test": "sui-studio test",
-    "test:watch": "sui-studio test --watch"
+    "test": "sui-studio test -- --coverage && npm run update:coverage",
+    "test:watch": "sui-studio test --watch",
+    "update:coverage": "node scripts/replace-coverage-badges.js && git add README.md"
```

Comment:
Should this script add and commit readme modifications?

Reply:
directly?

Reply:
what do you mean, the npm run script or the replace-coverage-badges?